docs/CONFIG-VARS.md

```diff
@@ -385,7 +385,7 @@ The EBS CSI driver is currently only used for kubernetes v1.23 or later AWS EKS
 | INGRESS_NGINX_NAMESPACE | NGINX Ingress Helm installation namespace | string | ingress-nginx | false | | baseline |
 | INGRESS_NGINX_CHART_URL | NGINX Ingress Helm chart URL | string | See [this document](https://kubernetes.github.io/ingress-nginx) for more information. | false | | baseline |
 | INGRESS_NGINX_CHART_NAME | NGINX Ingress Helm chart name | string | ingress-nginx | false | | baseline |
-| INGRESS_NGINX_CHART_VERSION | NGINX Ingress Helm chart version | string | "" | false | If left as "" (empty string), version 3.40.0 is used for Kubernetes clusters whose version is <= 1.21.X, and version 4.2.3 is used for Kubernetes clusters whose version is >= 1.22.X. | baseline |
+| INGRESS_NGINX_CHART_VERSION | NGINX Ingress Helm chart version | string | "" | false | If left as "" (empty string), version 3.40.0 is used for Kubernetes clusters whose version is <= 1.21.X, and version 4.3.0 is used for Kubernetes clusters whose version is >= 1.22.X. | baseline |
```

Comment:
Given the information here showing support for k8s 1.23-1.26 why are we not bumping the version on the helm chart to 4.5.2 (ingress-nginx version 1.6.4)? This is only a short-term fix and 4.3.0 was released on 10.22.22. We need to update to latest version each time we make a version change. Comment applies globally to this update.

Reply:
AWS still supports 1.22, if we move up the ingress-nginx version then DaC won't be able to support AWS 1.22 deployment. Hence the choice to use 4.3.0.